packages/kbn-config-schema/src/duration/index.ts

```diff
@@ -25,10 +25,14 @@ const timeFormatRegex = /^(0|[1-9][0-9]*)(ms|s|m|h|d|w|M|Y)$/;
 function stringToDuration(text: string) {
   const result = timeFormatRegex.exec(text);
   if (!result) {
-    throw new Error(
-      `Failed to parse [${text}] as time value. ` +
-        `Format must be <count>[ms|s|m|h|d|w|M|Y] (e.g. '70ms', '5s', '3d', '1Y')`
-    );
+    const number = Number(text);
+    if (typeof number !== 'number' || isNaN(number)) {
+      throw new Error(
+        `Failed to parse [${text}] as time value. ` +
+          `Format must be <count>[ms|s|m|h|d|w|M|Y] (e.g. '70ms', '5s', '3d', '1Y')`
+      );
```

Comment:
NIT: Error message no longer exhaustive regarding format. Should probably change to something like
```
Value must be a duration in millisecond or follow the format <count>[ms|s|m|h|d|w|M|Y] (e.g. '70ms', '5s', '3d', '1Y')
```

Reply:
I'll change the error message to this:

`Failed to parse [${text}] as time value. Value must be a duration in milliseconds, or follow the format <count>[ms|s|m|h|d|w|M|Y] (e.g. '70ms', '5s', '3d', '1Y'), where the duration is a safe positive integer.`